Makefile

```diff
@@ -6,7 +6,7 @@ test:
 
 coverage-html:
 	docker exec -it djangoplicity-newsletters coverage html
-	open ./htmlcov/index.html
+	google-chrome htmlcov/index.html
```

Comment:
Why google-chrome specifically?

Reply:
i did make this because in my computer not working the command open, for that reason i changed that command